src/Core/Misc/BoundedCounter.cs

```diff
@@ -49,7 +49,7 @@ public BoundedCounter(int upperBound)
         /// <summary>
         /// Attempt to increment Count
         /// </summary>
-        /// <returns>true iff we can increment without exceeding UpperBound</returns>
+        /// <returns>true if and only if we can increment without exceeding UpperBound</returns>
```

Comment:
Is [iff](https://en.wikipedia.org/wiki/If_and_only_if) the intent here? If so, I think expanding it out is clearer for TTS users at least...

Reply:
Reasonable. There are also 3 instances of `iff` in the Win32 project

Reply:
There are indeed, just splitting up the changes by namespace to ease review (see follow-up #741). Thanks for the quick reviews!